spec.bs

```diff
@@ -3426,27 +3432,23 @@ of the following global objects:
           |generatedBidIDL|, |ig|, |expectedCurrency|, |isComponentAuction|, and |global|'s
           [=InterestGroupBiddingScriptRunnerGlobalScope/group has ad components=].
         1. If |bidOutput| is not failure, then set |generatedBid| to |bidOutput|.
+    1. Let |bidDebugReportInfo| be a new [=bid debug reporting info=].
+    1. Set |bidDebugReportInfo|'s [=bid debug reporting info/interest group owner=] to
+       |ig|'s [=interest group/owner=]
     1. Let |debugLossReportUrl| be |global|'s
       [=InterestGroupBiddingAndScoringScriptRunnerGlobalScope/debug loss report url=] if it's not
       failure, null otherwise.
+    1. Set |bidDebugReportInfo|'s [=bid debug reporting info/bidder debug loss report url=] to |debugLossReportUrl|.
     1. If |generatedBid| is a [=generated bid=] and |generatedBid|'s [=generated bid/bid=]'s
       [=bid with currency/value=] &le; 0, then set |generatedBid| to null.
     1. If |generatedBid| is a [=generated bid=]:
       1. Let |debugWinReportUrl| be |global|'s
         [=InterestGroupBiddingAndScoringScriptRunnerGlobalScope/debug win report url=] if it's not
         failure, null otherwise.
       1. Set |generatedBid|'s [=generated bid/bid duration=] to |duration|,
-        [=generated bid/interest group=] to |ig|, [=generated bid/bidder debug loss report url=] to
-        |debugLossReportUrl|, [=generated bid/bidder debug win report url=] to |debugWinReportUrl|.
-    1. Otherwise:
-      1. Set |generatedBid| to a new [=generated bid=] with the following [=struct/items=]:
-        : [=generated bid/no bid=]
-        :: true
-        : [generated bid/interest group=]
-        :: |ig|
-        : [=generated bid/bidder debug loss report url=]
-        :: |debugLossReportUrl|
-    1. Return |generatedBid|.
+        [=generated bid/interest group=] to |ig|,
+      1. Set |bidDebugReportInfo|'s [=bid debug reporting info/bidder debug win report url=] to |debugWinReportUrl|.

```

Comment:
nit: we can inline |debugWinReportUrl| here, without the step declaring it above.

Reply:
Same as above. Also re-ordered it to be closer and ended other sentence in period rather than comma.